html/docs/files.html

```diff
@@ -95,6 +95,19 @@ <h4>Setup</h4>
               only affects your ability to <code>undo</code> changes.
             </p>
 
+						<p> 
```

Comment:
When adding this section, there should also be an entry `tags.data` after line 76